Comanche055/TVCSTROKETEST.agc

```diff
@@ -151,7 +151,7 @@ STRKCHK		INHINT
 		BZF	+3
 
 		CAE	ESTROKER	# BEGIN ON NEXT DAP PASS (PITCH OR YAW)
-		TS	STROKER		# (STROKING DONE IN PITCH ONLY, HOWEVER)
+		TS	STROKER		# (STROKING DONE IN PITCH ONLY. HOWEVER)
```

Comment:
Checking the scan, it looks like it could possibly be a comma. 
![981-comma](https://cloud.githubusercontent.com/assets/10581772/25561793/f2ae86ce-2d2f-11e7-857f-ec935e828a19.png)
I will change it back for now and commit all of these comma fixes.